<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="walia" ignored="0" name="Default test" tests="3" failures="1" timestamp="12 Aug 2018 23:27:49 GMT" time="120.986" errors="0">
  <testcase name="TestFullTransactionProcess" time="56.221" classname="AutomationFramework.TestCases"/>
  <testcase name="TestOnlyOneMagicMouseIsAdded" time="31.139" classname="AutomationFramework.TestCases"/>
  <testcase name="TestTransactionWithSameBillingAndShippingAddress" time="0.003" classname="AutomationFramework.TestCases">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	js cannot be resolved
	js cannot be resolved
	e cannot be resolved to a variable
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
js cannot be resolved
js cannot be resolved
e cannot be resolved to a variable

at AutomationFramework.TestCases.TestTransactionWithSameBillingAndShippingAddress(TestCases.java:243)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TestTransactionWithSameBillingAndShippingAddress -->
</testsuite> <!-- Default test -->
